.github/workflows/upstream_testing.yml

```diff
@@ -10,4 +10,4 @@ jobs:
     uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@develop"
     with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
       invoke_context_name: "NAUTOBOT_SECRETS_PROVIDERS"
-      plugin_name: "nautobot-secrets-provider"
+      plugin_name: "nautobot-plugin-secrets-providers"
```

Comment:
Want to make sure we're using the right name. I see in some places we have `-plugin` and in other places we don't.

Reply:
`plugin_name` input is used to composite Docker image name inside called action. Seems this input argument should be repository name, so the value provided here is correct as well as the template.

Reply:
Maybe we should fix the argument name to `repository-name`, to better reflect what's expected.